.github/workflows/codeql.yml

```
@@ -13,9 +13,9 @@ name: "CodeQL Advanced"
 
 on:
   push:
-    branches: [ "dev" ]
+    branches: [ "dev", "main" ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)